src/app/app.module.ts

```diff
@@ -38,6 +38,7 @@ import { StakingRouterModule } from '../staking-router-modules';
           autoLoadEntities: false,
           cache: { enabled: false },
           debug: false,
+          safe: true,
```

Comment:
What does state: true do?

Reply:
hmm, I think it's from investigating for the other task. removed.